### PR TITLE
Added the link to Linkedin

### PR DESCRIPTION
**Pull request by andreluiz93**
Please, take the link to my Linkedin page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
         <li>Alfredo Eduardo da Silva</li>
         <li>Allan Thomas Oliveira Franco</li>
         <li>André Luiz de Souza Almeida
```
Comment (edysegura) [inline index.html]: Please @andreluiz93  change it to have only one link! such as:

``` html
<a href="https://www.linkedin.com/in/andre-almeida-5a324995?trk=nav_responsive_tab_profile">André Luiz de Souza Almeida</a>
```

File: index.html [hunk 2]
```diff
@@ -9,7 +9,9 @@
     <ul>
         <li>Alfredo Eduardo da Silva</li>
         <li>Allan Thomas Oliveira Franco</li>
-        <li>André Luiz de Souza Almeida</li>
+        <li>André Luiz de Souza Almeida
```
Comment (edysegura) [inline index.html]: Please remove your name that is outside from the link

File: index.html [hunk 3]
```diff
@@ -10,6 +10,7 @@
         <li>Alfredo Eduardo da Silva</li>
         <li>Allan Thomas Oliveira Franco</li>
         <li>André Luiz de Souza Almeida</li>
```
Comment (edysegura) [inline index.html:12]: Hi @andreluiz93 ! Please remove the line above!

Comment (andreluiz93) [reply index.html:12]: Omg!!!

**Comment by edysegura** [comment]
Hi @andreluiz93! Please fix the conflicts so we can do the merge of your link!

**Comment by edysegura** [comment]
It is not valid anymore!

**Pull request closed without merging**